[CAS-999] Refac - MentionListView theme (#2036)
GetStream/stream-chat-android: UNRELEASED_CHANGELOG.md

            it.copy(threadsEnabled = false)
        }
```

> `MentionListView` now it is possible to customize background, empty state drawable and empty state label. It can be customisable with XML parameters and with theme.

Reviewer: ```suggestion
- `MentionListView` can be customisable with XML parameters and with a theme.
```
